src/game.py

```diff
@@ -0,0 +1,50 @@
+import sys
+from .players import AiPlayer, HumanPlayer
+from .board import Board
+
+class Game:
+
+    def __init__(self):
+        self.board = Board()
+        self.player1 = self.create_human_player()
+        self.player2 = self.create_ai_player()
+
+    def create_human_player(self):
+        while True:
+            name = input("Please choose X/O: ")
+            if name in ("X", "O", "x", "o"):
+                break
+            else:
+                while True:
+                    dec = input("Your input is not correct, really want to play? (y/n) ")
+                    if dec == 'y':
+                        break
+                    elif dec == 'n':
+                        sys.exit("Bye!")
+
+        return HumanPlayer(name.upper(), self.board)
+
+    def create_ai_player(self):
+        return AiPlayer(self.player1.opponent_piece(), self.board)
+
+    def check_board_state(self, win_flag=True):
+        if not self.board.possible_moves() and not self.board.is_winning():
+            sys.exit("Draw!")
+        elif self.board.is_winning() and win_flag:
```

Comment:
This may be kind of overengineered idea but game can have `ScoringStrategy` which given `board` decides the result: unsettled (game continues), p1, p2, draw.